### PR TITLE
[FEATURE] add status to header of ticket details

### PR DESCRIPTION
**Pull request by sarahmarciano**
Add the status of the ticket to the header
status

### CODE REVIEW

File: js/components/TicketDetails/Header.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { View } from 'react-native';
+import I18n from 'react-native-i18n';
+import { Text } from 'native-base';
+import { white, orange, green } from '../../utils/colors';
+
+function Header(props) {
```
Comment (raphaelboukara) [inline js/components/TicketDetails/Header.js]:
```js
function Header({ ticketNumber, status }) {
```

File: js/components/TicketDetails/Title.js [hunk 2]
```diff
@@ -7,9 +7,11 @@ import I18n from 'react-native-i18n';
 import moment from 'moment';
 import { grayBorder, primaryColor, white } from '../../utils/colors';
 
-
 function Title({ title, creationDate, text2, date2, assignedTo }) {
     function renderAssignedTo() {
+        if (assignedTo === '' || !assignedTo) {
+            return null;
+        }
         return (
```
Comment (raphaelboukara) [inline js/components/TicketDetails/Title.js]: no need, you can do:
```js
return assignedTo && (...
```

File: js/components/TicketDetails/index.js [hunk 3]
```diff
@@ -5,8 +5,24 @@ import Title from './TitleContainer';
 import Description from './DescriptionContainer';
 import FollowUps from './FollowUpsContainer';
 import Satisfactions from './SatisfactionsContainer';
+import Header from './Header';
+import { black, white } from '../../utils/colors';
 
 class TicketDetails extends Component {
+    static navigationOptions = ({ navigation }) => {
+        const { ticketNumber, status } = navigation.state.params;
```
Comment (raphaelboukara) [inline js/components/TicketDetails/index.js]: what the difference between ticketId and ticketNumber ?

File: js/components/Tickets/index.js [hunk 4]
```diff
@@ -40,8 +40,8 @@ class Tickets extends Component {
 
     renderItem({ item }) {
         const { navigate } = this.props.navigation;
-        const { ticketId, ticketNumber } = item;
-        return <TicketItem {...item} onPress={() => navigate('TicketDetails', { id: ticketId, ticketNumber })} />;
+        const { ticketId, ticketNumber, status } = item;
```
Comment (raphaelboukara) [inline js/components/Tickets/index.js]:
```js
const { navigate } = this.props.navigation,
  { ticketId, ticketNumber, status } = item;
```

File: js/components/Tickets/index.js [hunk 5]
```diff
@@ -40,8 +40,8 @@ class Tickets extends Component {
...
+        return <TicketItem {...item} onPress={() => navigate('TicketDetails', { id: ticketId, ticketNumber, status })} />;
```
Comment (raphaelboukara) [inline js/components/Tickets/index.js]: If you can access to `ticketNumber` and `status` via `ticketId` you only have to navigate to the screen `TicketDetails` with params `{ id: ticketId }`. This screen will contain a connected `Header` component that will receive `ticketId` as props.

File: js/components/TicketDetails/index.js [hunk 6]
```diff
@@ -5,8 +5,24 @@ import Title from './TitleContainer';
...
+        return ({
+            headerTitle: <Header ticketNumber={ticketNumber} status={status} />,
```
Comment (raphaelboukara) [inline js/components/TicketDetails/index.js]: `Header` has to be a connected component receiving `ticketId` as props. If `Header` is in fact `Ticket/Header` as `Header` will receive a props named `id`.

File: js/components/TicketDetails/Title.js (hunk 2 above)
Comment (sarahmarciano) [reply js/components/TicketDetails/Title.js]: Not working for me, here is the error I'm hitting
screenshot_1508827540

File: js/components/TicketDetails/index.js (hunk 3 above)
Comment (sarahmarciano) [reply js/components/TicketDetails/index.js]: ticket id is just the identifier of the ticket which they never show to the user, the ticket number is the ticket number which they show to the user,
actually, I don't really understand why do they need this ticket number, but these two props are different and have different purposes

File: js/components/Tickets/index.js (hunk 4 above)
Comment (sarahmarciano) [reply js/components/Tickets/index.js]: What do you mean?

**Pull request merged**